<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MercuryClass</class>
 <widget class="QMainWindow" name="MercuryClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mercury</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="newGameButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Start New Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadGameButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Mercury.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>MercuryClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
